susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1309,6 +1309,22 @@
         'BASECHANNEL' : 'opensuse_leap15_3-aarch64', 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15SP0SP1_SALT + PKGLIST15_X86_ARM,
         'DEST' : DOCUMENT_ROOT + '/pub/repositories/opensuse/15/3/bootstrap/'
     },
+    'openSUSE-Leap-15.4-x86_64' : {
```

Comment:
I guess we'll still need to adapt sumatoolbox and/or susemanger-sync-data later.

Reply:
why? Leap comes from SCC and should be included already.

Reply:
Heh, then maybe nothing is needed. Believe or not, I am still confused about what goes to sumatoolbox, and what goes to susemanager-sync-data :-P